packages/gluestick/src/commands/test/test.js

```diff
@@ -11,6 +11,22 @@ const JEST_PATH = `${require.resolve('jest').split('jest')[0]}.bin/jest`;
 const JEST_DEBUG_CONFIG_PATH = path.join(__dirname, 'jestEnvironmentNodeDebug.js');
 const TEST_MOCKS_PATH = `${path.join(__dirname)}`;
 
+const mergeCustomConfig = (defaultConfig: Object): Object => {
+  const customConfig: Object = require(path.join(process.cwd(), 'package.json')).jest;
+  if (!customConfig) {
+    return defaultConfig;
+  }
+
+  return Object.keys(customConfig).reduce((prev: Object, curr: string): Object => {
+    return {
+      ...prev,
+      [curr]: Array.isArray(customConfig[curr]) && Array.isArray(defaultConfig[curr])
+        ? defaultConfig[curr].concat(customConfig[curr])
+        : customConfig[curr],
+    };
+  }, defaultConfig);
+};
```

Comment:
Can we test this logic? It looks to me that `test` command requires finally a test.